web/html/src/manager/admin/payg-shared/info/payg-info-edit.tsx

```diff
@@ -46,6 +46,7 @@ const PaygInfoEdit = (props: Props) => {
           setItem(props.payg);
           setErrors(null);
         }}
+        disableEditing={!props.editing}
```

Comment:
Here and elsewhere where this name is used, I would recommend making the prop name consistent across the board. `payg.renderer.tsx` uses the name `readOnly` which seems to have the same purpose, I think that name is better.

Reply:
Fixed. Could please double check that I haven't missed any places where the property needs to be updated?